CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 ## Unreleased
 
+- Removed static methods from CoreCrypto that are globally available:
+
+  - removed `version()`
+  - removed `buildMetadata()`
+  - removed `setMaxLogLevel(level: CoreCryptoLogLevel)`
+  - removed `setLogger(logger: CoreCryptoLogger)`
+
+  Affected platforms: ios
+
+- Moved `historyClient(historySecret: HistorySecret)` into CoreCryptoClient Companion functions
+
+  Affected platforms: jvm, android
+
+- Renamed CoreCrypto wrapper to CoreCryptoClient and CoreCryptoProtocol to CoreCryptoClientProtocol
+
+  Affected platforms: ios
```

Comment:
I think at this point in the history you have not actually done this. Also we wanted not to do this.